src/app/routes/secrets/SecretsList.js

```diff
@@ -461,7 +466,7 @@ class SecretsList extends Component {
             </Tooltip>;
         } else if (requiresRequest) {
             const requestAccessLabel = isPending ? 'Cancel Request' : 'Request Access';
-            const requestType = isWrapped ? CONTROL_GROUP : STANDARD_REQUEST;
+            const requestType = isDynamicSecret ? DYNAMIC_REQUEST : isWrapped ? CONTROL_GROUP : STANDARD_REQUEST;
```

Comment:
Same as previous comment. See `NotificationsModal._getRequestTypeLabel`. In this case, a simple if/else provides better clarity.